hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -260,7 +260,8 @@ private boolean onDeltaSyncShutdown(boolean error) {
     public long sourceLimit = Long.MAX_VALUE;
 
     @Parameter(names = {"--op"}, description = "Takes one of these values : UPSERT (default), INSERT (use when input "
-        + "is purely new data/inserts to gain speed)", converter = OperationConverter.class)
+        + "is purely new data/inserts to gain speed) INSERT_OVERWRITE (use when input record can overwrite existing "
```

Comment:
We should also look at `DeltaSync::writeToSink` to trigger the insert_overwrite operation.